test/fixtures/users.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
delete this file, it's no need

Reply:
![Screenshot from 2021-09-01 21-03-55](https://user-images.githubusercontent.com/87750131/131676133-d69602f6-cb16-4130-86a9-ca8744b403c3.png)